resources/qml/TimelineView.qml

```diff
@@ -190,9 +190,26 @@ Page {
                     anchors.fill: parent
                     spacing: 0
 
-                    MessageView {
-                        Layout.fillWidth: true
-                        Layout.fillHeight: true
+                    StackLayout {
+                        id: stackLayout
+                        currentIndex: 0
```

Comment:
```suggestion
                        currentIndex: TimelineManager.onVideoCall ? 1 : 0
```

And remove the logic in the ActiveCallBar, I'd say.

Reply:
The idea is to be able to toggle the message view and the video call by clicking on the ActiveCallBar.

Reply:
Oh, that's better of course!